MonkeyBot/Services/Implementations/Trivia/OTDBTriviaInstance.cs

```diff
@@ -428,7 +427,7 @@ private async Task LoadQuestionsAsync(int count)
 
             if (!json.IsEmpty())
             {
-                OTDBResponse otdbResponse = await Task.Run(() => JsonConvert.DeserializeObject<OTDBResponse>(json)).ConfigureAwait(false);
+                OTDBResponse otdbResponse = await Task.Run(() => JsonSerializer.Deserialize<OTDBResponse>(json)).ConfigureAwait(false);
```

Comment:
Another thing: Although the serializer seems to automatically pick up the two converters it might make sense to explicitly add them to the Converters list in the SerializerOptions for clarity.